topics/admin/tutorials/reference-genomes/tutorial.md

```diff
@@ -0,0 +1,667 @@
+
+**Data managers to the rescue**
+
+Data Managers are a special class of Galaxy tool which allows for the download and/or creation of data that is stored within Tool Data Tables and their underlying flat location (e.g. .loc) files. These tools handle the creation of indices and the addition of entries/lines to the data table / .loc file via the Galaxy admin interface.
+
+Data Managers can be defined locally in the data manager and tool data table configuration files or installed through the Tool Shed. When Data Managers are installed from the Tool Shed, their configuration is added to the shed versions of the data manager configuration and tool data table configuration files.
+
+They are a flexible framework for adding reference data to Galaxy (not just genomic data). They are workflow compatible and can run via the Galaxy API. Data managers automatically update the appropriate location files when new data are installed.
+
+More detailed background information on data managers can be found [here](https://galaxyproject.org/admin/tools/data-managers/), and details on how to define a data manager for a tool can be found [here](https://galaxyproject.org/admin/tools/data-managers/how-to/define/). There is also discussion of this topic in [the Reference Genomes in Galaxy slides]({% link topics/admin/tutorials/reference-genomes/slides.html %}).
+
+> <comment-title>Pre-built data are available</comment-title>
+>
+> The usegalaxy.\* servers and Galaxy Community have a large amount of reference data online and available for use by your Galaxy server. For instructions on how to access and use these data, see [the Reference Data with CVMFS tutorial]({% link topics/admin/tutorials/cvmfs/tutorial.md %}).
+>
+> If your data are not available as part of the CVMFS repository, Galaxy Data Managers can be used to locally install and build reference data.
+>
+{: .comment}
+
+> <agenda-title></agenda-title>
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Galaxy Reference Data Components
+
+As discussed in the overview, Galaxy Data Managers handle the population of numerous Galaxy components. From the highest level to the lowest:
+
+- Addition of tool data table entries into the shed version of the tool data table configuration file, `shed_tool_data_table_conf.xml`. Multiple data tables with the same `name` attribute are automatically merged by Galaxy.
+- Population of the corresponding location (`.loc`) file when new reference data are installed. The `.loc` file used corresponds to the Data Manager and its specific version that was executed to install the data. The path to the location file can be found in the Data Manager's entry in the tool data table configuration file (above).
+- Generation of the reference data on disk, underneath the directory specified by `tool_data_path` in `galaxy.yml`.
+
+# Installing and Using Data Managers with the Galaxy UI
+
+## Install a the Fetch Genome Data Manager from the Tool Shed
```

Comment:
These always make me deeply uncomfortable for some strange reason and I write some useless blah blah to go here :sweat_smile: 

Good to not bother, it's unnecessary.

Reply:
You mean text in between headers?

Reply:
yeah just an aside.

Reply:
I don't disagree, haha